nservicebus/messaging/delayed-delivery.md

```diff
@@ -25,6 +36,13 @@ snippet:delayed-delivery-datetime
 
 In order to use message deferral the Timeout Manager feature must be enabled. When you defer a message, it is sent to the timeout manager requesting it to deliver the message at a later time. 
 
+snippet:configure-timeout-manager
+
+NOTE: In Version 4 Timeout Manager is enabled by default.
+
 NOTE: The sending endpoint must be running when the timeout is reached in order for the message to be sent. If the endpoint is not running when the timeout is reached then the message will be sent when the endpoint is next started. 
 
-NOTE: If you specify a negative timeout or a time that is in the past then the message will still be slightly delayed. The message will not be sent until the timeout manager has processed the request.
\ No newline at end of file
+NOTE: If you specify a negative timeout or a time that is in the past then the message will still be slightly delayed. The message will not be sent until the timeout manager has processed the request.
+
+## Persistance
+Delayed messages are being [persistence](/nservicebus/persistence/) in a location configured for `StorageType.Timeouts`. Those messages are stored there for the duration for which the delay was set.
```

Comment:
OK, why not add this snippet here? Even though it's not directly related to persistence, it'll save the reader some time for thinking and potentially searching through the docs.